warehouse/query-core/src/test/java/datawave/query/ExcerptTest.java

```diff
@@ -220,7 +220,7 @@ public void simpleTest() throws Exception {
         String queryString = "QUOTE:(farther) #EXCERPT_FIELDS(QUOTE/2)";
 
         // not sure why the timestamp and delete flag are present
-        Set<String> goodResults = Sets.newHashSet("HIT_EXCERPT:get much [farther] with a: : [] 9223372036854775807 false");
+        Set<String> goodResults = new HashSet<>(Collections.singleton("HIT_EXCERPT:get much [farther] with a: : [] 9223372036854775807 false"));
```

Comment:
Consider using `Set.of("string")` here and elsewhere.

Reply:
done